taworks/taform/models.py

```diff
@@ -60,7 +60,8 @@ class Application(models.Model):
     student = models.ForeignKey(Student, on_delete=models.CASCADE, blank=True)
     course = models.ForeignKey(Course, on_delete=models.CASCADE, blank=True)
     application_date = models.DateTimeField(auto_now_add=True)
-    preference = models.IntegerField(validators=[MaxValueValidator(3), MinValueValidator(0)])
+    STUDENT_PREFERENCE = ((1,'I prefer to TA this course'),(2,'I am able to TA this course'),(3,'I would prefer not to TA this course'),(0,'I am unable to TA this course'))
+    preference = models.CharField(max_length=50, choices=STUDENT_PREFERENCE)
```

Comment:
i think in the database we still want to store it as integerField for the algorithm.